package.json

```diff
@@ -18,12 +22,17 @@
     "grunt-contrib-qunit": "~0.3.0",
     "grunt-contrib-watch": "~0.5.3",
     "grunt-saucelabs": "~4.1.0",
-    "qunitjs": "^1.20.0"
+    "qunitjs": "^1.20.0",
+    "rollup": "^0.25.4",
+    "rollup-plugin-babel": "^2.4.0"
```

Comment:
Rollup is needed only for the `selector-set.global.js` build. It requires the module and immediately exports to `window.SelectorSet` inside an IIFE. If we drop the global support, the rollup dependency can be dropped too.